pyproject.toml

```diff
@@ -70,6 +71,9 @@ lint.ignore = [
     "E731",  # Do not assign a `lambda` expression, use a `def`
     "E741",  # Ambiguous variable name: `<VARIABLE>`
     "E743",  # Ambiguous function name: `<FUNCTION>`
+    # Rules from the stable "FURB" set that are ignored. See #27897
+    "FURB161",  # Use of `bin({existing}).count('1')`
```

Comment:
I think it would be worth noting that this is only for Python 3.9 support so it could be removed in future.